packages/contracts/src/test/Integration.t.sol

```diff
@@ -65,6 +62,13 @@ contract Integration is Test {
         game.commitSalt(COMMITTED_SALT);
         game.joinGame(gameID, 0, bytes(""), HAND_ROOT, DECK_ROOT, PROOF);
         vm.stopPrank();
+    }
+
+    function testGame() public {
+        uint256 gameID = 1;
+        // TODO
+        //game.createGame(2, game.allowAnyPlayerAndDeck);
```

Comment:
Let's delete these two lines (they were there because I had a "allowed to join game?" predicate that I removed because ethers.js couldn't easily encode it on the frontend, will read it later, but this TODO here is pretty useless).